doc/theory/glossary.tex

```diff
@@ -319,27 +319,27 @@
 	\begin{equation*}	
 	\lambda_k : \mathbb R \to \overline{\mathbb R}	
 	\end{equation*}	
-	defined by letting $\lambda_k$ be the $k$-th largest value of the set $\{\Lambda_i(t)\}_ {i \in I}$ where	
+	defined by letting $\lambda_k(t)$ be the $k$-th largest value of the set $\{\Lambda_i(t)\}_ {i \in I}$ where	
 	\begin{equation*}	
 	\Lambda_i(t) = \left[ \min \{t-b_i, d_i-t\}\right]_+	
 	\end{equation*}	
-	and $c_+ := \max(c,0)$. The function $\lambda_k$ is referred to as the \textit{$k$-layer of the persistence landscape}.	
+	and $c_+ := \max(c,0)$. The function $\lambda_k$ is referred to as the \textit{$k$-layer of the persistence landscape}.
 	
-	Intuitively, we can describe the set of graphs of a persistence landscape by first joining each of the points in the multiset to the diagonal via a horizontal as well as a vertical line, then clockwise rotating the figure 45 degrees and rescaling it by $1/\sqrt{2}$.	
+	We describe the graph of each $\lambda_k$ intuitively. For each $i \in I$, draw an isosceles triangle with base the interval $(b_i, d_i)$ on the horizontal $t$-axis, and sides with  slope  1  and $-1$.  This  subdivides  the  plane  into  a  number  of  polygonal regions.  Label each of these regions by the number of triangles containing it.  If $P_k$ is the union of the polygonal regions with values at least $k$, then the graph of $\lambda_k$ is the upper contour of $P_k$, with $\lambda_k(a) = 0$ if the vertical line $t=a$ does not intersect $P_k$.
 	
 	The persistence landscape construction defines a
 	%\hyperref[vectorization, kernel and amplitude]{vectorization}
 	vectorization of the set of persistence diagrams with target the vector space of real-valued function on $\mathbb N \times \mathbb R$. For any $p = 1,\dots,\infty$ we can restrict attention to persistence diagrams $D$ whose associated persistence landscape $\lambda$ is
 	%\hyperref[l^p norm]{$p$-integrable}
 	$p$-integrable, that is to say,	
-	\begin{equation} \label{equation: persistence landscape norm}	
+	\begin{equation} \label{equation:persistence_landscape_norm}	
 	||\lambda||_p = \left( \sum_{i \in \mathbb N} ||\lambda_i||^p_p \right)^{1/p}	
 	\end{equation}	
 	where
 	\begin{equation*}	
 	||\lambda_i||_p = \left( \int_{\mathbb R} \lambda_i^p(x)\, dx \right)^{1/p}	
 	\end{equation*}	
-	is finite. In this case we refer to (\ref{equation: persistence landscape norm}) as the
+	is finite. In this case we refer to (\eqref{equation:persistence_landscape_norm}) as the
```

Comment:
I would remove the parentheses, because eqref already produces them. In this case, in the pdf, you see ((1)).

Reply:
ok